db/init_db.js

```diff
@@ -1,31 +1,295 @@
-// code to build and initialize DB goes here
-const {
-  client
-  // other db methods 
-} = require('./index');
+
+const { client } = require('./client');
+const { createProduct } = require('./products')
+const { createUser } = require('./users')
+
+async function dropTables() {
+  console.log('Dropping All Tables..');
+  try {
+
+    await client.query(`
+        DROP TABLE IF EXISTS order_products;
+        DROP TABLE IF EXISTS orders;
+        DROP TABLE IF EXISTS products;
+        DROP TABLE IF EXISTS users
+      `);
+  } catch (error) {
+    throw error;
+  }
+}
 
 async function buildTables() {
+
+  console.log('Starting to build tables...');
   try {
-    client.connect();
 
-    // drop tables in correct order
+    await client.query(`
+        CREATE TABLE users(
+          id SERIAL PRIMARY KEY,
+          "firstName" VARCHAR(255) NOT NULL,
+          "lastName" VARCHAR(255) NOT NULL,
+          email VARCHAR(255) UNIQUE NOT NULL,
+          "imageURL" VARCHAR(255) DEFAULT 'imageUrl',
```

Comment:
Insert an actual default?

Reply:
Might want to add `NOT NULL` constraint.  That way if `null` is insterted, then the default is used.